test/features/test_user_can_delete_skill.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../test_helper'
+
+class UserDeletesSkillTest < FeatureTest
+  include TestHelpers
+
+  def test_user_can_delete_a_skill
+    skill_1 = {:title => "Green Bay Packers",
+               :description => "Best team in the NFL"}
+    skill_2 = {:title => "Chicago Bears",
+               :description => "The Bears still suck"}
```

Comment:
We like to prepare the "minimum data necessary to run the test". In this test we only need one skill to make sure we can delete a skill